lib/kernel/doc/src/code.xml

```diff
@@ -614,7 +617,11 @@ ok = code:finish_loading(Prepared),
       <desc>
         <p>Tries to load any modules not already loaded in the list
 	<c><anno>Modules</anno></c> in the same way as
-          <seemfa marker="#load_file/1">load_file/1</seemfa>.</p>
+          <seemfa marker="#load_file/1">load_file/1</seemfa>.
+          Opposite to <seemfa marker="#ensure_loaded/1">ensure_loaded/1</seemfa>,
+          concurrent calls to this function will attempt to load
+          modules concurrently. Furthermore, modules are loaded
```

Comment:
If find this sentence confusing.

Does that mean that if two processes call this function "at the same time", one or more of the modules can become loaded twice?

Reply:
It is more that the modules will be read and processed more than once but only loaded once. Should we replace "will attempt to load" by "will read and prepare modules concurrently, but load them once"?

Reply:
In principle there's nothing that prevents us from avoiding this multiple-load issue the same way it's avoided for `ensure_loaded/1` - it's just that when I implemented this mechanism for `enusre_loaded/1` the multi-load scenario was so complex I couldn't implement the change it in a reasonable way, so I decided to skip it.